Merge Sorted Array.py

```diff
@@ -0,0 +1,19 @@
+# Time Complexity: O(n+m)
+# Space Complexity: O(1)
+# Did this code successfully run on Leetcode : yes
+# Any problem you faced while coding this : was not able to solve edge cases like 
+# [2,0]
```

Comment:
The best approach for identifying the edge cases would be to think the opposite of the happy case and null cases. Check what's changes are important in problem statement